adaptivefiltering/segmentation.py

```diff
@@ -88,14 +88,30 @@ def save(self, filename):
     def show(self):
         """Create a new InteractiveMap with bounds from the segmentation.
 
-        :param grid:
-             The grid object which holds the map and the right side interface
-        :type grid: ipyleaflet.grid
+        :
         """
 
         segmentation_map = Map(segmentation=self)
         return segmentation_map.show()
 
+    def convert(self, srs_in, srs_out="EPSG:4326"):
```

Comment:
Having this as a user facing API suggests that there is a use case where a user needs this. When would that be?

Reply:
Okay, so we remove that.